processor/transformprocessor/README.md

```diff
@@ -14,157 +14,144 @@
 <!-- end autogenerated section -->
 
 > [!NOTE]
-> This documentation applies only to version `0.120.0` and later. For information on earlier versions, please refer to the previous [documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/release/0.119.x/processor/transformprocessor/README.md).
+> This documentation applies only to version `0.120.0` and later. **Configuration from previous version is still supported**, but no longer documented in this README. For information on earlier versions, please refer to the previous [documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/release/0.119.x/processor/transformprocessor/README.md).
 
-The transform processor modifies telemetry based on configuration using the [OpenTelemetry Transformation Language](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl).
+The Transform Processor modifies telemetry based on configuration using the [OpenTelemetry Transformation Language](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl) (OTTL).
 
-For each signal type, the processor takes a list of conditions and statements associated to a [Context type](#contexts) and executes the conditions and statements against the incoming telemetry in the order specified in the config.
-Each condition and statement can access and transform telemetry using functions and allow the use of a condition to help decide whether the function should be executed.
+For each signal type, the processor takes a list of statements and executes them against the incoming telemetry, following the order specified in the configuration.
+Each statement can access and transform telemetry using functions, and allows the use of a condition to help decide whether the function should be executed.
 
+- [Writing OTTL](#writing-ottl)
```

Comment:
The more I read through this doc the more important it seems to learn how to write OTTL before even worrying about configuring the processor. Switched this and the `Config` section to help new users find `Writing OTTL` first.

Reply:
Could we link out to a section on writing OTTL inside the OTTL docs instead? I'd like for the transform processor readme to cut to the chase and show how it's configured, and I think the callouts should be enough of an attention getter to help new users find out how to write OTTL.

Reply:
Should I move the entire `Writing OTTL` section to `pkg/ottl/README`?

Reply:
That's what I was thinking. It would also group things a bit better in my opinion, since all OTTL-specific documentation would be in the OTTL readme.

Reply:
Ok I'll make that change.

Reply:
Updated